### ISSUE TITLE
[StatusIndicator]: separate from StatusBadge

### ISSUE THREAD

**Issue by mart-sessman** [TEDI-Design-System/design#232]
It is used more widely than just in StatusBadge. The usage should be documented aswell, e.g. use only with other components or text.

**Comment by karolin-kivisikk** [comment]
Separated from status badge: https://www.figma.com/design/jWiRIXhHRxwVdMSimKX2FF/TEDI-READY-2.45.70?node-id=48720-166708&t=JB46lWnLynhVwfHW-4

**Comment by AngelicaSinisalu** [comment]
Vastab WCAG nõuetele, ainult mõned märkused erandjuhtude kohta: https://www.figma.com/design/jWiRIXhHRxwVdMSimKX2FF?node-id=48720-166709#1768343387
